data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+-- =======================================================================
+--                 Texts for the Atlantean Mission 2
+-- =======================================================================
+
+include "scripting/richtext_scenarios.lua"
+
+function jundlina(title, text)
+   return speech("map:princess.png", "2F9131", title, text)
+end
+function loftomor(text)
+   return speech("map:loftomor.png", "FDD53D", "Loftomor", text)
+end
+function sidolus(text)
+   return speech("map:sidolus.png", "FF1A30", "Sidolus", text)
+end
+function colionder(text)
+   return speech("map:colionder.png", "33A9FD", "Colionder", text)
+end
+function opol(text)
+   return speech("map:opol.png", "FF3FFB", "Opol", text)
+end
+function ostur(text)
+   return speech("map:ostur.png", "375FFC", "Ostur", text)
+end
+function kalitath(text)
+   return speech("map:kalitath.png", "FF0000", "Kalitath", text)
+end
+
+
+-- =======================================================================
+--                                Objectives
+-- =======================================================================
+
+obj_find_port = {
+   name = "obj_find_port",
+   title = _ "Find a place to build a port",
+   number = 1,
+   body = objective_text(_"Find a portspace",
+   p(_[[To set foot on this island we first must find a building space where a port can be built. Explore the nearby coast line to find such a building plot.]]) ..
+   p(_[[After the port is build we should cancel the remaining ships expedition, to unload its wares as well.]]))
+}
+
+obj_basic_infrastructure = {
+   name = "obj_basic_infrastructure",
+   title = _ "Ensure the supply of build wares",
+   number = 4,
+   body = objective_text(_"Establish a basic production",
+   p(_[[We will need at least a quarry, a few lumberjacks and a sawmill. A forester might or might not be needed depending on how fast the trees are growing around you. Find the balance.]]))
+}
+
+obj_tools = {
+   name = "obj_tools",
+   title = _ "Build tools quickly",
+   number = 1,
+   body = objective_text(_"Make new tools",
+      p(_[[Most of our tools have been lost in the ship tragedy. Our people will not be able to fulfill their duties without proper tools. Luckily we still have some iron ore and plenty of coal. Build a production of tools, it is a prerequisite for success on island. Don’t forget to build a smelter as well.]]))
+}
+
+obj_mining = {
+   name = "obj_mining",
+   title = _ "Explore the mountains for ressources",
```

Comment:
'Resources' is spelt with one 's'. Also, it's either "Explore the mountains to find resources" or "Search the mountains for resources", but you can't "explore for" something.